src/cache-service.js

```diff
@@ -0,0 +1,87 @@
+const path = require('path');
+const fs = require('fs');
+const constants = require('./constants');
+
+/**
+ * A File Cache controller for store and retrieve files from disk
+ *
+ * @class CacheService
+ */
+class CacheService {
+    constructor() {
+        this.cachePath = path.join(constants.DATABASE, 'cache');
```

Comment:
Instead of doing this constants.DATABASE thing, you should make the db a parameter to the constructor.